README.md

```diff
@@ -9,29 +9,30 @@ This is my research and reflection journal for DGL 104 winter 2023
 - want to create some new things with learned technology or skill.
 - want to use this in an organization i am going to work and get better in using those new technology.
 - perfect in what i am learning in this course.
-- 
 
 
 ## interests
 
 - *Get* a nice job and make satisfied with my job.
 - up to date with the new technologies.
-- I want to learn Java language, HTML and CSS scripting languages for my  future goal, where these are source to make a website and run's in a effective manner.
+- I want to learn Java language, HTML and CSS scripting languages for my future goal, where these are source to make a website and run's in a effective manner.
 - want to develop one site for my village and local area about things in the places.
 
 ## Come up
 
 - develop web app or mobile app for my region on places. It helps in outer people know about particular place.
 - Work through the knowledge you have keep track on it.
 - Make sure to revise what you know and what has to do.
-- i am working on my programming skills and language where it helps me in the project  and some other courses, by spending some time on the outcomes of this course.
+- I am working on my programming skills and language where it helps me in the project  and some other courses, by spending some time on the outcomes of this course.
 
```

Comment:
I think what you're saying here is that the work you do here and in other courses all together helps you become a better programmer. I think that's true, and that's a good observation to make.

Are there _specific_ things that you've learned here or in other courses that you think has helped you recently?